<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BCI framework - Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Theme:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_light">
          <property name="text">
           <string>Light</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_dark">
          <property name="text">
           <string>Dark</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_user_directory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Projects directory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_projects_directory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Records directory:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_records_directory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QWidget" name="widget_restart" native="true">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Some changes will take efect after restart&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Danger zone</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="class" stdset="0">
       <string>fill_background</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="horizontalSpacing">
          <number>30</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_restore_projects">
           <property name="text">
            <string>Restore default projects</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_reset_connections">
           <property name="text">
            <string>Reset connections</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_reset_projects">
           <property name="text">
            <string>Reset user projecs</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_remove_records">
           <property name="text">
            <string>Remove all records</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_reset_montages">
           <property name="text">
            <string>Reset montages</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_reset_notebooks">
           <property name="text">
            <string>Reset notebooks</string>
           </property>
           <property name="class" stdset="0">
            <string>danger</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_restore_projects</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset_projects</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_remove_records</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset_connections</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset_montages</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_light</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_dark</sender>
   <signal>clicked()</signal>
   <receiver>widget_restart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_user_directory</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
